ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -142,7 +142,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     echo -e "[DEBUG] GST_PLUGIN_PATH is '$GST_PLUGIN_PATH'"
 
     # nnstreamer env variables and include paths
-    export NNSTREAMER_CONF=$NNST_ROOT
+    export NNSTREAMER_CONF=$NNST_ROOT/nnstreamer.ini
```

Comment:
What is a goal of this file (e.g., ..../nnstreamer.ini)?

Reply:
.ini file includes the path for sub-plugins. I updated this commit to remove unnecessary env values about sub-plugins path.

Reply:
It seems that the updated contents is better. Does it mean that  sub-plugins will be maintained by nnstreamer.ini file (without  environment variables such as the NNSTREAMER_****)?

Reply:
yes, installed ini file includes each sub-plugin paths.
```
in meson build script
subplugin_install_prefix = join_paths(nnstreamer_prefix, 'lib', 'nnstreamer')
nnstreamer_conf.set('SUBPLUGIN_INSTALL_PREFIX', subplugin_install_prefix)
```
```
in nnstreamer.ini.in
[filter]
filters=@SUBPLUGIN_INSTALL_PREFIX@/filters/
customfilters=@SUBPLUGIN_INSTALL_PREFIX@/customfilters/

[decoder]
decoders=@SUBPLUGIN_INSTALL_PREFIX@/decoders/
```

Reply:
Acked-by : Geunsik Lim.